recipes/podofo/all/conanfile.py

```diff
@@ -58,6 +59,8 @@ def _configure_cmake(self):
         self._cmake = CMake(self)
         self._cmake.definitions["PODOFO_BUILD_LIB_ONLY"] = True
         self._cmake.definitions["PODOFO_BUILD_SHARED"] = self.options.shared
+        if tools.Version("0.9.7") <= self.version and not self.settings.compiler.cppstd:
+            self._cmake.definitions["CMAKE_CXX_STANDARD"] = 11
```

Comment:
Could we add a check is it's less then 11 since it wont compile... nice to let the use know?